scripts/create-release-archive.js

```diff
@@ -0,0 +1,193 @@
+const child_process = require('child_process') // eslint-disable-line camelcase
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const packageJsonScriptsInclude = [
+  'start',
+  'build',
+  'serve'
+]
+
+function usage () {
```

Comment:
we might want to consider an args library like commander or yargs

Reply:
Considered it, but I felt that this was quicker than trying to decide which one to use 😅

Reply:
I'm impressed with the simplicity of this approach - I'd always just gone with an args library before but I couldn't see any issues with this and it's pretty simple to read/maintain.